ProjFS.Mac/PrjFSKext/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -144,7 +144,19 @@ VirtualizationRootHandle VirtualizationRoot_FindForVnode(
     
     VirtualizationRootHandle rootHandle = RootHandle_None;
     
-    vnode_get(vnode);
+    errno_t error = vnode_get(vnode);
+    if (error != 0)
+    {
+        const char* name = vnode_getname(vnode);
+        mount_t mount = vnode_mount(vnode);
+        vfsstatfs* vfsStat = mount != nullptr ? vfs_statfs(mount) : nullptr;
+        KextLog_FileNote(vnode, "VirtualizationRoot_FindForVnode: vnode_get() failed (error = %d) on vnode we'd expect to be live; name = '%s', recycling = %s, mount point mounted at path '%s'", error, name ?: "[NULL]", vnode_isrecycled(vnode) ? "yes" : "no", vfsStat ? vfsStat->f_mntonname : "[NULL]");
```

Comment:
How severe do you think this scenario is?  Should we consider making it Info or Error level?

Reply:
Same question applies other spots that `KextLog_FileNote` is called.  How did you decide what was severe enough to warrant logging at the `Error` level?

Reply:
Sorry about that, looks like I didn't subject this series of commits to my usual level of scrutiny before opening the PR; probably because they're originally from the #555 branch which is at more of a prototype-level quality. I think they can all be errors, so I'll go through and change them shortly.

Incidentally, "note" is above "info" in severity; I've mapped it to `os_log`'s "default" level. For consistency's sake, we should probably rename our note level too, and deprecate the old name.